content/news/2024-07-26-shiny-phyloseq-on-EU/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Shiny phyloseq interactive tool available on usegalaxy.eu - practical outcome of the Bioconductor Hackathon'
+date: '2024-26-07'
+tease: "We are thrilled to announce, that thanks to the help of the Bioconductor folks, we could deploy the Shiny phyloseq App as an interactive tool (IT) on usegalaxy.eu"
+hide_tease: false
+authors: Paul Zierep 
+subsites: [all-eu, global]
+---
+
+# Shiny phyloseq Interactive tool (IT)
+
+[Shiny apps](https://shiny.posit.co/) are web apps using R functionality, that give easy responsive access to R packages. 
+Interactive tools are a great way to work with data interactively and responsive using Galaxy. In theory all shiny apps could become ITs, but so far the 
+wrapping and deployment of shiny apps as ITs was technically challenging. Thanks to a Hackathon together with folks from the Bioconductor 
+communities best-practices for the development of ITs using Shiny apps were developed, focusing mainly on the development of a docker image, that can be used
```

Comment:
I would be interested to know more about how you construct the docker image. Is there a link that you could add here / share with me?